spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -185,7 +192,7 @@ public AuditRecordController auditController(
 	}
 
 	@Bean
-	@ConditionalOnBean({ StreamDefinitionRepository.class })
+	@ConditionalOnStreamsEnabled
```

Comment:
Nice improvement. 
Perhaps we should group all ConditionalOnStreamsEnabled beans in a common inner class and annotate only that class.

Reply:
@tzolov yeah, that would be nice! Let's do this in a different story. I will create one for it.